order-summary/run.js

```diff
@@ -68,38 +66,48 @@ function getFlowItemFromVariant(variant_ids) {
  *  beyond this comment.
  */
 
+ /** Helper function to break total number of Variant IDs into 100 ID chunks */
+async function getFlowItemFromVariant(variantIDs) {
+  const apiCallPromises = []
+  for (let i = 0; i < variantIDs.length; i += 100) {
+    apiCallPromises.push(makeFlowAPICall(variantIDs.slice(i, i + 100)))
+  }
+  const flowObjects = await Promise.all(apiCallPromises)
+  return flowObjects.flat()
+}
+
 /** Loop through all the files in the input directory and parse order. */
-fs.readdir("./samples", function (err, files) {
+fs.readdir(`${inputDirName}`, function (err, files) {
```

Comment:
you can just pass `inputDirName` instead of doing string interpolation

Reply:
😐can imagine why i did that